tests/unit/memefficiency.tcl

```diff
@@ -148,12 +149,13 @@ start_server {tags {"defrag external:skip"} overrides {appendonly yes auto-aof-r
                 # make sure the defragger did enough work to keep the fragmentation low during loading.
                 # we cannot check that it went all the way down, since we don't wait for full defrag cycle to complete.
                 assert {$frag < 1.4}
-                # since the AOF contains simple (fast) SET commands (and the cron during loading runs every 1000 commands),
```

Comment:
Changed this partly for accuracy, the number was changed from 1000 to 1024 in the MP-AOF PR, partly because you might want to update this comment to mention this change